turingschool-examples/whats-cookin-starter-kit
# Feature/fav recipe icon nav

**What’s this PR do?**

- From user story favorite page the user is able to click on the favorite recipe icon and be directed to the user story view recipe page
- 
**How should this be manually tested?**

-  pull down PR and input npm start in terminal and open http://http://localhost:8080/ in browser
How should this be manually tested?

**Where should the reviewer start?**
- Click on a recipe on the side bar or recipe icon on main page
- Save a recipe as a favorite
- click the fav icon on the right side of header bar (heart image) to go to the favorites page and view fav recipes
- click on the icon to view the recipe information

**Any background context you want to provide?**

- Needs refactor to dry up code

Screenshots (if appropriate)
![2022-07-17 18 43 51](https://user-images.githubusercontent.com/102189342/179431688-2d93267f-03a6-4126-9ab1-4973c8ccb09d.gif)
